src/js/sensor_helpers.js

```diff
@@ -19,19 +19,23 @@ export function have_sensor(sensors_detected, sensor_code) {
     return false;
 }
 
-export function sensor_status(sensors_detected) {
+export function sensor_status(sensors_detected, gps_fix_state) {
```

Comment:
This way we don't need to provide 0 values :)
```suggestion
export function sensor_status(sensors_detected = 0, gps_fix_state = 0) {
```